src/js/main.js

```diff
@@ -212,26 +212,27 @@
                 constant.textFont,
                 function (response) {
                     roomModel.font = response;
```

Comment:
changed the for of loops to regular loops. they were causing issues with the webpack build